go-controller/pkg/ovn/egressip_test.go

```diff
@@ -6354,18 +6495,19 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 
 				expectedNatLogicalPort1 := "k8s-node1"
 
-				nodeIPsASdbIDs := getEgressIPAddrSetDbIDs(NodeIPAddrSetName, DefaultNetworkControllerName)
-				fakeOvn.asf.EventuallyExpectAddressSetWithAddresses(nodeIPsASdbIDs, []string{node1IPv4})
-
-				egressSvcPodsV4, _ := addressset.GetHashNamesForAS(egresssvc.GetEgressServiceAddrSetDbIDs(DefaultNetworkControllerName))
-				egressipPodsV4, _ := addressset.GetHashNamesForAS(getEgressIPAddrSetDbIDs(EgressIPServedPodsAddrSetName, DefaultNetworkControllerName))
-				nodeIPsV4, _ := addressset.GetHashNamesForAS(nodeIPsASdbIDs)
 				nodeSwitch.QOSRules = []string{"egressip-QoS-UUID"}
+
+				namespaceAddressSetv4, _ := buildNamespaceAddressSets(eipNamespace, []string{"10.128.0.3"})
```

Comment:
nit: not a fan of the hardcoding here and in some other places.. is the 0.3 same as egressPodIP or a random IP we are using?
if this is my only comment then I won't compel a repush.

Reply:
took another look, this happens because the test author did not specify an address for the pod, this is the next address assigned. Luckily they grabbed the address from the pod so I used that instead of the hardcode